src/shared/api/apiClient.ts

```
@@ -0,0 +1,80 @@
+// axios 인스턴스 생성
+export const apiClient: AxiosInstance = axios.create({
+	baseURL: API_BASE_URL,
+	timeout: 10000, // 10초 타임아웃
+	headers: {
+		'Content-Type': 'application/json',
+	},
+});
+
+// 요청 인터셉터
+apiClient.interceptors.request.use(
+	(config: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)